docs/index.rst

```diff
@@ -9,12 +9,13 @@
 Welcome to RestrictedPython's documentation!
 ============================================
 
-.. include:: idea.rst
```

Comment:
If I see that correctly, `idea.rst` was included twice, and is now still reachable via the toctree, isn't it?

Reply:
Right.